app/assets/javascripts/components/meetings_list.es6.jsx

```diff
@@ -0,0 +1,68 @@
+class MeetingsList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.perPage = 12;
+    this.meetings = [];
```

Comment:
I already have a `meetings` on that state containing all meetings. The instance variable is used to store the current silce of meetings.